packages/Python/lldbsuite/test/lang/swift/dwarfimporter/C/TestSwiftDWARFImporterC.py

```diff
@@ -58,6 +59,7 @@ def test_dwarf_importer(self):
         target.Clear()
         lldb.SBDebugger.MemoryPressureDetected()
 
+    @skipIf(archs=['ppc64le'])
```

Comment:
If these tests don't work, can we have SR tracking them?

Reply:
The following SR has already been created to track these failures:-
https://bugs.swift.org/browse/SR-10082